toolkit/src/main/resources/com/google/api/codegen/nodejs/sample.snip

```
@@ -164,14 +164,16 @@
 @end
 
 @private authFunc_ADC(class)
-  function {@class.auth.authFuncName}(callback) {
-    google.auth.getClient({
+  async function {@class.auth.authFuncName}(callback) {
+    const auth = new google.auth.GoogleAuth({
       scopes: ['https://www.googleapis.com/auth/cloud-platform']
-    }).then(client => {
-      callback(client);
-    }).catch(err => {
-      console.error('authentication failed: ', err);
     });
+    try {
```

Thinking we shouldn't wrap with the try/catch here, and just let it throw.